### ISSUE TITLE
Uncaught TypeError: Cannot read property 'toLowerCase' of undefined

### ISSUE THREAD

**Issue by raulk** [danielbrodin/atom-project-manager#261]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.10.4
**Thrown From**: [project-manager](https://github.com/danielbrodin/atom-project-manager) package, v3.2.0
### Stack Trace

Uncaught TypeError: Cannot read property 'toLowerCase' of undefined

```
At /Users/raul/.atom/packages/project-manager/lib/Manager.js:22

TypeError: Cannot read property 'toLowerCase' of undefined
    at /Users/raul/.atom/packages/project-manager/lib/Manager.js:22:47
    at ObservableArray.find (/Users/raul/.atom/packages/project-manager/node_modules/mobx/lib/mobx.js:1698:27)
    at Manager.addProject (/Users/raul/.atom/packages/project-manager/lib/Manager.js:21:40)
    at executeAction (/Users/raul/.atom/packages/project-manager/node_modules/mobx/lib/mobx.js:593:19)
    at Manager.res (/Users/raul/.atom/packages/project-manager/node_modules/mobx/lib/mobx.js:565:16)
    at Manager.saveProject (/Users/raul/.atom/packages/project-manager/lib/Manager.js:132:10)
    at EditView.saveProject (/Users/raul/.atom/packages/project-manager/lib/views/EditView.js:37:13)
    at HTMLDivElement.<anonymous> (/Users/raul/.atom/packages/project-manager/lib/views/EditView.js:19:14)
```
### Commands

```
     -2:19.5.0 command-palette:toggle (atom-text-editor.editor.mini.is-focused)
  2x -2:16.7.0 core:move-down (atom-text-editor.editor.mini.is-focused)
  2x -2:14.7.0 core:move-up (atom-text-editor.editor.mini.is-focused)
     -2:13.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:13.9.0 project-manager:edit-project (atom-text-editor.editor.mini)
     -1:41.6.0 core:focus-previous (input.input-text)
  3x -1:34.3.0 core:cancel (atom-pane.pane.active)
     -1:09.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:49.1.0 command-palette:toggle (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:46 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:46 project-manager:save-project (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:39.4.0 application:add-project-folder (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:04.7.0 command-palette:toggle (ul.list-inline.tab-bar.inset-panel)
     -0:02.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:02.9.0 project-manager:save-project (ul.list-inline.tab-bar.inset-panel)
     -0:01.0 core:focus-next (input.input-text)
```
### Config

``` json
{
  "core": {},
  "project-manager": {
    "prettifyTitle": false
  }
}
```
### Installed Packages

``` coffee
# User
atom-typescript, v10.1.6 (active)
autocomplete-go, v1.1.1 (active)
builder-go, v1.0.1 (active)
file-icons, v1.7.19 (active)
git-plus, v5.17.0 (inactive)
git-time-machine, v1.5.3 (inactive)
go-config, v1.2.3 (active)
go-debug, v0.1.22 (active)
go-get, v1.0.4 (active)
go-oracle, v0.2.1 (inactive)
go-plus, v4.2.0 (active)
go-rename, v0.3.0 (active)
godoc, v0.4.0 (active)
gofmt, v1.1.8 (active)
gometalinter-linter, v1.1.1 (active)
gorename, v1.0.3 (active)
highlight-line, v0.11.1 (active)
highlight-selected, v0.11.2 (active)
linter, v1.11.16 (active)
merge-conflicts, v1.4.4 (active)
minimap, v4.25.0 (active)
navigator-godef, v1.0.4 (active)
pigments, v0.34.0 (active)
project-manager, v3.2.0 (active)
Remote-FTP, v0.8.9 (active)
terminal-plus, v0.14.5 (active)
tester-go, v1.0.6 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (active)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (active)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (active)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by danielbrodin** [comment]
Have you tried to save a project without any paths?

**Comment by edomato** [comment]
I had the same issue today.

What I did was to try to save the project with "ProjectManager: Save project" from the command palette.

The dialog screen that appears doesn't have a path field.

atom-project-manager is at last version (3.2.5).

Greets.

**Comment by edomato** [comment]
More information :-)

I tried a fresh install of Atom (moved .atom to .atom-bak) and only installed atom-project-manager.

I opened a directory with the whole project I'm working on and when I tried to save the project with command palette, it was saved without error and now my .atom directory has the corresponding projects.cson file with the title of the project and the corresponding path.

So, maybe another package is conflicting with atom-prohect-manager?.

Any suggestion?
